### ISSUE TITLE
Sidebar: obey the user

### ISSUE THREAD

**Issue by sujato** [suttacentral/suttacentral#1446]
As a general rule, avoid second-guessing the user. Unless its really necessary, don't do anything that the user does not explicitly ask for.

The sidebar should only extend or retract on explicit request by the user. It should be closed by default at all screen widths, and only open or close when the user tells it to. 

The only exception to this is on narrow screens, where it is essential that the sidebar close in order to read the content. Thus once clicking something on the sidebar, it should retract (as it does currently).

To sum up:

- Sidebar should be always retracted by default.
- It is only opened by explicit user request (either clicking the hamburger menu or swiping the screen from the left).
- On wider screens, it is only closed by user request.
- On narrow screen, it closes automatically when an item is clicked.

**Comment by ihongda** [comment]
![Peek 2019-07-06 00-31](https://user-images.githubusercontent.com/4037966/60735321-77bf4000-9f85-11e9-96f9-aaa691d22274.gif)

**Comment by sujato** [comment]
The updated version still does not meet requirements. If you have a narrow screen and expand it horizontally, the sidebar emerges automatically. This must never happen under any circumstances. The sidebar **only** emerges when explicitly requested by the user, either by swiping or by clicking the hamburger icon.

**Comment by ihongda** [comment]
![Peek 2019-08-09 23-30](https://user-images.githubusercontent.com/4037966/62790527-e69c3580-bafd-11e9-8c72-b976448badcf.gif)

**Comment by sujato** [comment]
Great, that looks good.
